zookeeper/core/component.py

```diff
@@ -543,34 +512,33 @@ def component(cls: Type):
     return cls
 
 
-def configure(
+def recursively_configure_component_instance(
```

Comment:
I think this name is a bit misleading, since it isn't actually a recursive method? As far as I can see it's called once by `configure` and then returns.

Reply:
This is a good point, it was originally recursive but now is not. I've renamed it in 7741227.